src/Model/Table/AclNodesTable.php

```diff
@@ -154,7 +154,7 @@ public function node($ref = null)
             foreach ($ref as $key => $val) {
                 if (strpos($key, $type) !== 0 && strpos($key, '.') === false) {
                     unset($ref[$key]);
-                    $ref["{$type}0.{$key}"] = $val;
+                    $ref[$type . '0.' . $key . ((is_null($val)) ? ' IS' : '')] = $val;
```

Comment:
Ah, awesome. Wasn't aware of that, nice. Is there any standard for these quotes, when we have to use a single or double quotes?